notebooks/community/ml_ops/stage7/get_started_with_model_monitoring_xgboost.ipynb

```diff
@@ -0,0 +1,1477 @@
+{
```

Comment:
<p>Could we replace <code>import google.cloud.aiplatform as aiplatform</code> with <code>from google.cloud import aiplatform</code> for consistency? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1211/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>done</p>